pootle/static/js/browser/components/BrowserTableRow.js

```diff
@@ -81,6 +82,13 @@ const BrowserTableRow = React.createClass({
     if (itemType === ITEM_PROJECT) itemTypeName = 'project';
     if (itemType === ITEM_LANGUAGE) itemTypeName = 'language';
 
+    const progressBar = total === 0 ? '' :
```

Comment:
Unless this is intentional, note that in order to display _nothing_, `null`, `false` or `undefined` should be used instead — `''` will create an unnecessary span/text elements otherwise. See http://stackoverflow.com/questions/30097091/correct-way-to-define-an-empty-dom-element-in-react for more details.